README.md

```diff
@@ -2,11 +2,858 @@
 
-The first section will walk you through span creation and propagation using the OpenTelemetry Java API and [Spring's RestTemplate Http Web Client](https://spring.io/guides/gs/consuming-rest/). This approach will use the "vanilla" OpenTelemetry API to make explicit tracing calls within an application's controller. 
+The [first section](#section-1-manual-instrumentation-with-java-sdk) will walk you through span creation and propagation using the OpenTelemetry Java API and [Spring's RestTemplate Http Web Client](https://spring.io/guides/gs/consuming-rest/). This approach will use the "vanilla" OpenTelemetry API to make explicit tracing calls within an application's controller. 
+
+The [second section](#section-2-using-spring-handlers-and-interceptors) will build on the first. It will walk you through implementing spring-web handler and interceptor interfaces to create traces with minimal changes to existing application code. Using the OpenTelemetry API, this approach involves copy and pasting files and a significant amount of manual configurations. 
+
+The [third section](#section-3-instrumentation-using-opentelemetry-contrib-spring-in-progress) will walk you through the annotations and configurations defined in the opentelemetry-contrib-spring package. This section will equip you with new tools to streamline the step up and instrumentation of OpenTelemetry on Spring and Spring Boot applications. With these tools you will be able to setup distributed tracing with little to no changes to existing configurations and easily customize traces with minor additions to application code.  
+
+# Manual Instrumentation Guide
+
+In this guide we will be using a running example. In section one and two, we will create two spring web services using Spring Boot. We will then trace the requests between these services using OpenTelemetry. Finally, in section three we will explore tools in the opentelemetry-contrib-spring package which can improve this process.
+
+## Create two Spring Projects
+
+Using the [spring project initializer](https://start.spring.io/), we will create two spring projects.  Name one project FirstService and the other SecondService. Make sure to select maven, Spring Boot 2.3, Java, and add the spring-web dependency. After downloading the two projects include the OpenTelemetry dependencies and configuration listed below. 
+
+## Setup for Section 1 and Section 2
+
+Add the dependencies below to enable OpenTelemetry in FirstService and SecondService. The Jaeger and LoggerExporter packages are recommended for exporting traces but are not required for this section. As of May 2020, Jaeger, Zipkin, OTLP, and Logging exporters are supported by opentelemetry-java. Feel free to use whatever exporter you are most comfortable with. 
```

Comment:
[Suggestion] I think I mentioned that before, use $name$ to get a code font.

Reply:
Which terms should I put in code font?

Reply:
I'm sorry, I meant backticks (` `).
Internally at Google we use them to demarcate anything that is a code item rather than natural langauge, so in this case, I'd have `FirstService` and `SecondService`. At least internally, these would render differently as sans serif fixed width code words.